src/main/kotlin/org/wfanet/measurement/common/Flow.kt

```diff
@@ -132,16 +133,16 @@ private class SingleConsumedFlowItem<T>(singleItem: T) : ConsumedFlowItem<T>() {
 }
 
 /**
- * Consumes the first item of the [Flow], producing a [Flow] for the remaining
- * items.
+ * Consumes the first item of the [Flow], producing a [Flow] for the remaining items.
  *
- * Note that this starts a new coroutine in a separate [CoroutineScope] to
- * produce the returned [Flow] items using a [Channel]. As a result, the
- * returned [Flow] is hot.
+ * Note that this starts a new coroutine in a separate [CoroutineScope] to produce the returned
+ * [Flow] items using a [Channel]. As a result, the returned [Flow] is hot.
  *
  * @return a [ConsumedFlowItem] containing the first item and the [Flow] of
+ * ```
```

Comment:
This looks wrong here. I think it's assuming this is code when it's supposed to be a continuation of the `@return`.

Reply:
Will fix. Not sure why my regex missed this.

Reply:
Done.